bcda/auth/models.go

```diff
@@ -49,3 +51,17 @@ func (t *Token) AfterFind() error {
 	}
 	return err
 }
+
+func GetACOByClientID(clientID string) (models.ACO, error) {
```

Comment:
I like that this is moved to a plugin-neutral location.

Reply:
thanks, i do too.  i figured since it is a db select, the models would be a good place for it